src/prefect/tasks/core/collections.py

```diff
@@ -131,12 +131,19 @@ class Dict(Task):
     def __init__(self, *args: Any, **kwargs: Any):
         super().__init__(*args, **kwargs)
 
-    def run(self, **task_results: Any) -> dict:  # type: ignore
+    def run(self, keys: Iterable[Any], values: Iterable[Any]) -> dict:  # type: ignore
```

Comment:
Though I don't expect any issues, it might be worthwhile to call this out as a breaking change

Reply:
fair fair

Reply:
👍 updated